integration/tests/verification/suite.go

```diff
@@ -34,27 +34,25 @@ type Suite struct {
 
 // Ghost returns a client to interact with the Ghost node on testnet.
 func (s *Suite) Ghost() *client.GhostClient {
-	ghost := s.net.ContainerByID(s.ghostID)
-	cli, err := lib.GetGhostClient(ghost)
+	client, err := s.net.ContainerByID(s.ghostID).GhostClient()
 	require.NoError(s.T(), err, "could not get ghost client")
-	return cli
+	return client
 }
 
 // AccessClient returns a client to interact with the access node api on testnet.
 func (s *Suite) AccessClient() *testnet.Client {
-	chain := s.net.Root().Header.ChainID.Chain()
-	cli, err := testnet.NewClient(fmt.Sprintf(":%s", s.net.AccessPorts[testnet.AccessNodeAPIPort]), chain)
+	client, err := s.net.ContainerByName("access_1").TestnetClient()
```

Comment:
can you define a constant for `"access_1"` instead of using magic strings everywhere?